src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-CSharp/Properties/launchSettings.json

```diff
@@ -19,9 +19,9 @@
     "Company.WebApplication1": {
       "commandName": "Project",
       "launchBrowser": true,
+      "applicationUrl": "https://localhost:5001;http://localhost:5000",
```

Comment:
Can you also remove this? 
https://github.com/aspnet/templating/pull/390/files#diff-bd9b64bdb7d7d33da368a9efbf4d1cfaL16

It should not be necessary anymore (try it out), I think @danroth27 failed to mention it in the bug.